packages/odyssey-react-mui/src/ui-shell/UiShell/UiShell.tsx

```diff
@@ -98,16 +101,23 @@ const UiShell = ({
       >
         <ErrorBoundary fallback={appComponent} onError={onError}>
           <CssBaseline />
-
-          <UiShellContent
-            {...componentProps}
-            appBackgroundContrastMode={appBackgroundContrastMode}
-            appComponent={appComponent}
-            hasStandardAppContentPadding={hasStandardAppContentPadding}
-            initialVisibleSections={initialVisibleSections}
-            onError={onError}
-            optionalComponents={optionalComponents}
-          />
+          <UiShellColorsProvider
+            contrastMode={appBackgroundContrastMode}
+            sideNavBackgroundColor={
+              componentProps?.sideNavProps?.mainBackgroundColor ||
+              odysseyDesignTokens.HueNeutralWhite
```

Comment:
opinion: should SideNav handle the fallback/default color internally rather than UiShell passing it?

Reply:
No strong feelings on that one. I can move the default to the provider